pints/_nested/_nestedEllipsoid.py

```diff
@@ -0,0 +1,313 @@
+#
+# Exponential natural evolution strategy optimizer: xNES
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+# Some code in this file was adapted from Myokit (see http://myokit.org)
+#
+import pints
+import numpy as np
+import scipy
+import scipy.linalg
+import multiprocessing
```

Comment:
Don't see this used anywhere! (There are classes/methods in pints to evaluate things in parallel/sequentially via a shared interface btw, see http://pints.readthedocs.io/en/latest/function_evaluation.html)